lib/bosh/workspace/release.rb

```diff
@@ -54,12 +55,17 @@ def release_dir
       @path ? File.join(@repo_dir, @path) : @repo_dir
     end
 
-    private
-
     def repo
       @repo ||= Rugged::Repository.new(repo_dir)
     end
 
+    private
```

Comment:
Was needed to get the tests working correctly. The release object's repo attribute (which was previously private) was the one receiving the checkout/checkout_tree calls, and I needed a way to expose that object to expect().. expect_any_instance_of was failing since it only allows any one instance to receive that message.